src/main/java/seedu/address/model/person/passenger/Price.java

```diff
@@ -13,16 +12,15 @@
     public static final String MESSAGE_CONSTRAINTS =
             "The price you are willing to pay should be entered as a positive decimal number.";
     public static final String VALIDATION_REGEX = "\\d+(\\.\\d\\d?)?";
-    public final String value;
+    public final Double value;
 
     /**
      * Constructs a {@code Price}.
      *
      * @param price A valid price.
      */
-    public Price(String price) {
-        requireNonNull(price);
-        checkArgument(isValidPrice(price), MESSAGE_CONSTRAINTS);
+    public Price(double price) {
```

Comment:
maybe keeping the requireNonNull might be useful since that's what we're encouraged to do this week

Reply:
Good point. I removed it as Intellij suggested doing so as double data types aren't nullable but I'll add it in just in case.